src/server/main.go

```diff
@@ -32,46 +31,47 @@ func getPort() string {
 	return "8080"
 }
 
+// Check if env variable exists, if not set to default value, return value of env variable
+func checkEnv(key string, defaultVal string) string {
+	if os.Getenv(key) == "" {
+		os.Setenv(key, defaultVal)
```

Comment:
I'd say don't bother setting the env vars when using the default.  We should only be checking these on startup anyway.